R/limits.r

```diff
@@ -109,11 +109,11 @@ ylim <- function(...) {
 #' ggplot2:::limits(c("A", "b", "c"), "x")
 #' ggplot2:::limits(c("A", "b", "c"), "fill")
 #' ggplot2:::limits(as.Date(c("2008-01-01", "2009-01-01")), "x")
-limits <- function(lims, var) UseMethod("limits")
+limits <- function(lims, var, ...) UseMethod("limits")
```

Comment:
I think you'd be better off making an explicit `call` argument since otherwise `...` will swallow any extra arguments.